File: JS_Problems/Migratory_Birds.js
/*
 * Complete the 'migratoryBirds' function below.
 *
 * The function is expected to return an INTEGER.
 * The function accepts INTEGER_ARRAY arr as parameter.
 */

function migratoryBirds(arr) {
  for (let i = 0; i < arr.length; i++) {
    let count = new Array(6).fill(0);
    for (let bird of arr) {
      count[bird]++;
    }
    return count.indexOf(Math.max(...count));
  }
}

File: JS_Problems/migratory_Birds.py
# Complete the 'migratoryBirds' function below.
#
# The function is expected to return an INTEGER.
# The function accepts INTEGER_ARRAY arr as parameter.
#

def migratoryBirds(arr):
    for i in range(0,len(arr)):
        count=[0]*6
        for bird in arr:
            count[bird]+=1
        return count.index(max(count)) 